.travis.yml

```diff
@@ -72,9 +72,7 @@ script:
     - if [ $ZFS_BUILD_TAGS = 0 ]; then
         export FIO_SRCDIR=$PWD/../fio;
         travis_wait 5 sudo bash ./tests/cbtest/script/test_uzfs.sh -T all
-	echo $?
```

Comment:
FYI:-
In travis each command in script is processed by a special bash function. This function manipulates $? to produce logs suitable for display. Consequently, you should not rely on the value of $? in script section to alter the build behavior.
https://github.com/travis-ci/travis-ci/issues/3771

Reply:
good to know this.. Thanks